<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tela_configurar_legenda_estoque</class>
 <widget class="QDialog" name="tela_configurar_legenda_estoque">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>154</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>410</width>
    <height>154</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>410</width>
    <height>154</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Configurar Legenda</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="alterar_valores">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>130</width>
     <height>115</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="title">
    <string>Aterar valores</string>
   </property>
   <widget class="QLineEdit" name="le_ideal">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>85</y>
      <width>25</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="inputMask">
     <string>000; </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_zerado">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>25</y>
      <width>25</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="inputMask">
     <string>000; </string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lb_zerado">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>25</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Estoque zerado:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_normal">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>65</y>
      <width>25</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="inputMask">
     <string>000; </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_minimo">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>45</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Estoque mínimo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_normal">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>65</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Estoque normal:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_ideal">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>85</y>
      <width>78</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Estoque ideal:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_minimo">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>45</y>
      <width>25</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="inputMask">
     <string>000; </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Alterar_cor">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>5</y>
     <width>265</width>
     <height>115</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="title">
    <string>Alterar Cores</string>
   </property>
   <widget class="QPushButton" name="btn_alterar_cor_zerado">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>25</y>
      <width>125</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Alterar cor (Zerado)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_alterar_cor_normal">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>65</y>
      <width>125</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Alterar cor (Normal)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_alterar_cor_ideal">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>85</y>
      <width>125</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Alterar cor (Ideal)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_alterar_cor_minimo">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>45</y>
      <width>125</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Alterar cor (Mínimo)</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="Pre_visualizar">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>130</width>
      <height>100</height>
     </rect>
    </property>
    <property name="title">
     <string>pré-visualizar</string>
    </property>
    <widget class="QTableWidget" name="tw_legenda">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>15</y>
       <width>120</width>
       <height>78</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>80</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoScroll">
      <bool>false</bool>
     </property>
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="rowCount">
      <number>4</number>
     </property>
     <property name="columnCount">
      <number>1</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>19</number>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <column/>
     <item row="0" column="0">
      <property name="text">
       <string>Zerado</string>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="background">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>150</green>
         <blue>150</blue>
        </color>
       </brush>
      </property>
      <property name="foreground">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>Mínimo</string>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="background">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>210</green>
         <blue>210</blue>
        </color>
       </brush>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>Nomal</string>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="background">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>204</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </property>
     </item>
     <item row="3" column="0">
      <property name="text">
       <string>Ideal</string>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="background">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>253</green>
         <blue>126</blue>
        </color>
       </brush>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>411</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>35</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>35</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QPushButton" name="btn_confirmar">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>24</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Confirmar</string>
      </property>
      <property name="icon">
       <iconset resource="imagens.qrc">
        <normaloff>:/img/img/1345902916_camera_test.png</normaloff>:/img/img/1345902916_camera_test.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_cancelar">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>24</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Cancelar</string>
      </property>
      <property name="icon">
       <iconset resource="imagens.qrc">
        <normaloff>:/img/img/1345902747_button_cancel.png</normaloff>:/img/img/1345902747_button_cancel.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>18</width>
        <height>18</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_voltar_padrao">
      <property name="minimumSize">
       <size>
        <width>151</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>151</width>
        <height>24</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>Configuração padrão</string>
      </property>
      <property name="icon">
       <iconset resource="imagens.qrc">
        <normaloff>:/img/img/reset.png</normaloff>:/img/img/reset.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>18</width>
        <height>18</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>le_zerado</tabstop>
  <tabstop>le_normal</tabstop>
  <tabstop>le_minimo</tabstop>
  <tabstop>le_ideal</tabstop>
  <tabstop>btn_alterar_cor_zerado</tabstop>
  <tabstop>btn_alterar_cor_minimo</tabstop>
  <tabstop>btn_alterar_cor_normal</tabstop>
  <tabstop>btn_alterar_cor_ideal</tabstop>
  <tabstop>btn_confirmar</tabstop>
  <tabstop>btn_cancelar</tabstop>
  <tabstop>btn_voltar_padrao</tabstop>
  <tabstop>tw_legenda</tabstop>
 </tabstops>
 <resources>
  <include location="imagens.qrc"/>
 </resources>
 <connections/>
</ui>
